v5/api/cpp/vision.rst

```diff
@@ -960,6 +1111,25 @@ that can be detected by the Vision Sensor
  pros::E_VISION_OBJECT_LINE         Object returned is a line type.
 ================================== ====================================================================================================
 
+pros::vision_zero_e_t
+---------------------
+
+This enumeration defines different zero points for returned vision objects.
+
+::
+
+  typedef enum vision_zero {
+    E_VISION_ZERO_TOPLEFT = 0,
+    E_VISION_ZERO_CENTER = 1
+  } vision_zero_e_t;
+
+======================= =============================================
+ Value
+======================= =============================================
+ E_VISION_ZERO_TOPLEFT   (0,0) coordinate is the top left of the FOV
+ E_VISION_ZERO_CENTER    (0,0) coordinate is the center of the FOV

```

Comment:
This shouldn't be `(0, 0)`

Reply:
are you saying it should be `(0, 0)`? I made them all `(0,0)` (no space) to be consistent with what's in the header files

Reply:
Nope, I misunderstood the purpose of the enum.